pkg/json/ingest.go

```diff
@@ -37,7 +37,8 @@ func IngestStream(ctx *ls.Context, baseID string, input io.Reader, parser Parser
 	if err != nil {
 		return nil, err
 	}
-	root, err := ls.Ingest(builder, pd)
+	ing := ls.Ingester{Schema: parser.Layer}
```

Comment:
Change this function to get the ingester as an arg. It is not used the cache now.